src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -5,44 +5,89 @@
 package frc.robot.subsystems;
 
 import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.TalonFXInvertType;
+import com.ctre.phoenix.motorcontrol.can.SlotConfiguration;
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
- 
+import frc.robot.Constants;
 
 public class DriveTrain extends SubsystemBase {
 
-  private TalonFX frontL; 
-  private TalonFX frontR; 
-  private TalonFX backL; 
+  private TalonFX frontR;
+  private TalonFX frontL;
   private TalonFX backR;
+  private TalonFX backL;
```

Comment:
nit: these fields should be marked final
```suggestion
  private final TalonFX frontR;
  private final TalonFX frontL;
  private final TalonFX backR;
  private final TalonFX backL;
```